.gitignore

```diff
@@ -46,3 +46,6 @@ cypress/videos
 cypress/plugins/index.js
 cypress/plugins/cy-ts-preprocessor.js
 cypress/tsconfig.json
+
+# VsCode
+.vscode/settings.json
```

Comment:
There is already an entry for the vscode files a bit higher. I'd just remove the exclusions `!` as we shouldn't be committing any IDE related files to the repo. Simply remove all `!.vscode/xxx` entries and the `.vscode/*` should already cover whole directory.

Reply:
Yeah, we've covered this on Slack already.